pkg/limayaml/defaults.go

```diff
@@ -551,9 +562,24 @@ func FillDefault(y, d, o *LimaYAML, filePath string) {
 		y.MountType = o.MountType
 	}
 	if y.MountType == nil || *y.MountType == "" {
-		if *y.VMType == VZ {
+		switch *y.VMType {
+		case VZ:
 			y.MountType = ptr.Of(VIRTIOFS)
-		} else {
+		case QEMU:
+			y.MountType = ptr.Of(NINEP)
+			// Use REVSSHFS if the instance was created with Lima prior to v1.0
+			if _, err := os.Stat(filePath); err == nil { // existing instance
```

Comment:
There is a more complex `isExistingInstanceDir` function in #1951 that should probably be used here too.

Reply:
done